src/components/views/Instance/Edit/Deployment.tsx

```diff
@@ -82,6 +83,24 @@ export function Deployment(): JSX.Element {
             setCompileJobsTotalPages(response.payload.totalPages);
             setCompileJobsPage(page);
             setCompileJobs(response.payload.content);
+
+            // also grab the job on the next page if possible
+            if (
+                response.payload.content.length == compileJobsPageSize &&
+                response.payload.content[response.payload.content.length - 1].id > 1
+            ) {
+                const extraResponse = await DreamMakerClient.getCompileJob(
+                    instanceEditContext.instance.id,
+                    response.payload.content[response.payload.content.length - 1].id - 1
```

Comment:
This isn't a valid way to get the job on the next page because ID is a database ID and not guaranteed to be sequential.

You'd have to actually fetch the next page using the pagination API.